compiler/q-implant/src/QImplant.cpp

```diff
@@ -282,4 +285,38 @@ void QImplant::write(loco::Graph *g)
   }
 }
 
+void QImplant::forward_qparam(loco::Graph *g)
+{
+  std::set<luci::CircleOpcode> forwardable_opcode{
+    luci::CircleOpcode::RESHAPE, luci::CircleOpcode::SPLIT, luci::CircleOpcode::TRANSPOSE};
+
+  auto forwardable = [&forwardable_opcode](luci::CircleOpcode opcode) {
+    return forwardable_opcode.find(opcode) != forwardable_opcode.end();
+  };
+
+  for (auto node : loco::postorder_traversal(loco::output_nodes(g)))
+  {
+    auto circle_node = loco::must_cast<luci::CircleNode *>(node);
+    // skip when node is output
+    if (circle_node->opcode() == luci::CircleOpcode::CIRCLEOUTPUT)
+      continue;
```

Comment:
This code seems unnecessary.

Reply:
Output nodes have no successors, so they will be ignored in the below loop.

Reply:
>Output nodes have no successors, so they will be ignored in the below loop.

I missed that.
I will reflect it.

Thank you